README.md

```diff
@@ -124,7 +124,41 @@ where `type` is the category of the change, `description` is a short sentence to
 
 For more information, please see [here](https://github.com/lob/generate-changelog#usage).
 
+## To 2.0.0
+
+The rasa-x-helm chart in version 2.0.0 supports using an external Rasa OSS deployment.
+
+### Enabling an external Rasa OSS deployment
+
+The rasa-x-helm chart >= 2.0.0 supports an option to use an external Rasa OSS deployment.
+Below you can find an example of configuration that uses the external deployment.
```

Comment:
Should we make a note of how they need to configure either event broker to get it properly forwarding?

Reply:
Good point! I'll add it